### DISCUSSION TITLE
Input tipo "Recommended"

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#862] [Proposals]
Para empezar a dar un poco de flexibilidad a las OBS en el CMS nos ha surgido la necesidad de hacer opcionales algunos campos que antes eran mandatory, el problema es que al quitar el mandatory se pone el optional y necesitamos poder "recomendar" al usuario algunos campos aunque sean opcionales.
¿Qué opináis?¿Se os ocurre alguna otra opción?
<img width="868" alt="image" src="https://user-images.githubusercontent.com/95479574/197721541-c1cf2bbe-ca7b-4cfe-a29b-0d8722fb7e07.png">

**Comment by yceballost** [comment]
Hemos estado hablando sobre esto y creemos que tiene algo de dark pattern recomendar rellenar una info que es opcional. El opcional cubre la necesidad que planteas y el usuario siempre es el que toma esa decisión sin influenciarle.

Tampoco creemos que deberíamos abrir la posibilidad de que se le pase un label custom al "opcional" porque generar diferentes formas de llamar a un campo opcional es algo inconsistente y no aporta (resta más bien) a la ux.

**Reply by MontseTelefonica** [reply]
Entiendo lo que dices, y comparto. Esta necesidad nace de no querer hacer mas IFs en desarrollo para cada OB y que el mismo CMS pueda servir para hacer las distintas versiones pero pudiendo tener una opción "default" que nosotros queremos potenciar.

**Reply by yceballost** [reply]
Podrías usar el text helper para hacer esa recomendación, no?

**Reply by MontseTelefonica** [reply]
Si, optaré por probar usar el helper text para esto!
